randovania/games/samus_returns/logic_database/Area 7.txt

```diff
@@ -163,7 +157,7 @@ Extra - asset_id: collision_camera_005
       All of the following:
           Grapple Beam
           Any of the following:
-              Power Bomb ≥ 2 and Lay Power Bomb
+              Power Bomb ≥ 2
               All of the following:
                   Lay Bomb
                   Plasma Beam or Shoot Beam Burst
```

Comment:
This should be fulfilled with Grapple and 2 Power Bombs but is missing Morph Ball to actually use the Power Bombs.

Reply:
Good catch. Though I guess technically this is fine since you'd need Morph to get here anyway.